src/style.css

```diff
@@ -338,10 +338,22 @@ button {
   cursor: pointer;
 }
 
+button.red {
+  background-color: red;
+}
+
 button:hover {
   background-color: #3e8e41;
 }
 
+button.red:hover {
+  background-color: darkred;
+}
+
+.text-center {
```

Comment:
**suggestion:** did you check the spectre CSS classes? I am pretty sure one exist for this: https://picturepan2.github.io/spectre/utilities/text.html#text